### DISCUSSION TITLE
[Question] Option to group NavigationRailDestinations/menu points? Examples attached.

### DISCUSSION THREAD

**Discussion by tobsome** [flet-dev/flet#3059] [Q&A]
Hey guys,

quick question: Is there a way to make a dropdown in my "menu"? I built it with NavigationRailDestinations and would love to group my entries to free up some space in future. I "designed" examples what I mean with that with an arrow in front of the menu point.

Dropdown closed:
![dropdown_closed](https://github.com/flet-dev/flet/assets/50978173/fa78e149-ae3e-4bcd-ae9a-4eb591ef7cdc)

Dropdown opened:
![dropdown_opened](https://github.com/flet-dev/flet/assets/50978173/eb492bf8-fc3a-4da3-83cf-9afb865e2af2)

Any idea how I could reach my goal? Thanks a lot for any answer! :)

**Comment by abdokhalil07** [comment]
if i understood your question correctly
i think you try to use expansiontile
https://flet.dev/docs/controls/expansiontile
![close](https://github.com/flet-dev/flet/assets/94837888/af1b56dc-e236-48da-b4ea-9467543d0445)
![open](https://github.com/flet-dev/flet/assets/94837888/e8ced321-8ae4-4b11-8686-8da08c2e17b3)
